### PR TITLE
chore: add contributing guide

### PR DESCRIPTION
**Pull request by AntonKozachuk**
## What/Why/How?
Add OpenAPI CLI Contributing Guide.

## Reference
None.

## Testing
N/a

## Screenshots (optional)
N/a

## Check yourself

- [x] Code is linted
- [ ] Tested with redoc/reference-docs/workflows
- [ ] All new/updated code is covered with tests

## Security

- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+2. Run `npm install` in the repository root.
+3. If you’ve fixed a bug or added code that should be tested, don't forget to add tests!
+4. Ensure the test suite passes (`npm test`). Tip: `npm test -- --watch TestName` is helpful in development.
+5. Format your code with prettier (`npm run prettier`).
+
+## Development Setup
+
+You will need [Node.js](http://nodejs.org) at `v12.0.0+`.
+
+After cloning the repo, run:
```
Comment (skadinna) [inline .github/CONTRIBUTING.md:32]: question: should this be forking or cloning? We advise them to fork the repo above, so maybe we should stick to that advice :)

Comment (AntonKozachuk) [reply .github/CONTRIBUTING.md:32]: Makes sense, will change now.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+There are some other scripts available in the `scripts` section of the `package.json` file.
+
+## Project Structure
+
+- **`__mocks__`**: contains basic mocks for e2e tests.
+  
+- **`__tests__`**: contains e2e tests. The e2e tests are written and run with [Jest](https://jestjs.io/).
+
+- **`docs`**: contains the docs and guides, also can be found [here](https://redoc.ly/docs/cli/).
```
Comment (skadinna) [inline .github/CONTRIBUTING.md:79]:
```suggestion
- **`docs`**: contains the documentation source files. When changes to the documentation are merged, they automatically get published on the [Redocly docs website](https://redoc.ly/docs/cli/).
```

**Review by skadinna — APPROVED** [review]
This looks good! :tada: I left a few small comments.

**Comment by skadinna** [comment]
oh and I think @adamaltman mentioned you should include a note about following the Code of Conduct :thinking:

**Comment by AntonKozachuk** [comment]
> oh and I think @adamaltman mentioned you should include a note about following the Code of Conduct 🤔

Then I should probably add the Code of Conduct file itself, right?

**Comment by AntonKozachuk** [comment]
> oh and I think @adamaltman mentioned you should include a note about following the Code of Conduct 🤔

Added.

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
+# OpenAPI CLI Contributing Guide
```
Comment (Marshevskyy) [inline .github/CONTRIBUTING.md:1]: please add a link to this file to the main README.md

Comment (AntonKozachuk) [reply .github/CONTRIBUTING.md:1]: Added.

**Pull request merged**